pkg/oauth/server/osinserver/registrystorage/storage.go

```diff
@@ -248,6 +255,20 @@ func (s *storage) convertToAccessToken(data *osin.AccessData) (*oauthapi.OAuthAc
 	if err := s.user.ConvertToAccessToken(data.UserData, token); err != nil {
 		return nil, err
 	}
+
+	// Check if the optional inactivity Timeout needs to be set
+	var timeoutsin int32 = 0
+	wrapper := data.Client.(*clientWrapper)
```

Comment:
Can you replace this cast with the code from my first comment?

Reply:
done